src/state/useUserState.ts

```diff
@@ -271,6 +272,12 @@ export function useUserState(props: {
           });
         }
       },
+      setWhereFound(whereFound: string){
+        dispatch({
+          type: "WHERE_FOUND",
+          whereFound,
+        })
+      },
```

Comment:
is there a reducer that actually handles this action? I didn't see it in the diff.

Ie. there should be a function like:

```ts
function reduceWhereFound({config: { whereFound }}: UserState, action: UserStateAction){
  if(action.type === "WHERE_FOUND") return action.whereFound;
  else return whereFound;
}
```